### ISSUE TITLE
Implementation of new hosting solution

### ISSUE THREAD

**Issue by daine** [cds-snc/design-gc-conception#549]
As a developer, I want to have a CI/CD pipeline for the design system's documentation website in AWS to make deployment and maintenance easier and faster.

**TODO**
- [x] #570
- [x] GitHub actions
- [x] #571

**Outcomes**
- [x] https://github.com/cds-snc/gcds-docs runs on AWS

**Deliverables**
- [ ] An accessible link to the documentation website (could be an amplify link for now)

**Estimated Effort**
- [ ] XS: 1 to 2 hours
- [ ] S: half a day
- [ ] M: 1-2 day
- [x] L: 1 week
- [ ] XL: Move to an epic

**Out of scope**
- Moving domains - not until we're ready to switch
- Retiring old domains

**Comment by daine** [comment]
Related to: https://github.com/cds-snc/platform-core-services/issues/476

**Comment by patheard** [comment]
I'll start work this week by getting the [cds-snc/gcds-terraform repo](https://github.com/cds-snc/gcds-terraform) ready to hold the Terraform for the static site.  This will include:

- Creating the top level folders to hold the site's TF resources.
- Updating the Terraform plan/apply workflows to be aware of the new top level folders.
- Creating a `design.cdssandbox.xyz` Hosted Zone for the new site's temporary URL (used for testing until it's ready for launch).
- Updating any existing CDN resources to that they don't cause a conflict with new static site resources.

**Comment by patheard** [comment]
Initial folder setup is now done.  I've also hooked up the subdomain delegation so we should be good to start creating DNS records and validating ACM certificates.

**Comment by ClementineHahn** [comment]
Due to reduced team capacity this EPIC are still on going

**Comment by daine** [comment]
Domains are currently now accessible through:
https://design-system.cdssandbox.xyz/
https://systeme-design.alpha.canada.ca/

Next step on the Epic is:
- https://github.com/cds-snc/design-gc-conception/issues/591

**Comment by daine** [comment]
Decision record: https://docs.google.com/document/d/1Pv6stEuQyU_z3eYi4gqd8sbGKr9qQptZsPg9f3UiYkM/edit#heading=h.l5nt9a4k9ygg
